### DISCUSSION TITLE
Is there a way to build a long/short strategy? not long only?

### DISCUSSION THREAD

**Discussion by dkoleary88** [gbeced/basana#33] [Q&A] [answered]
Is there a way to build a long/short strategy? not long only?
Seems like the examples show the Strategy publishing a BUY or SELL signal only (long only).
Is there a way to create a Strategy that can do LONG, SHORT and CLOSE signals? 
i.e. SHORT and CLOSE are different because criteria for shorting a pair is different to the criteria for closing and existing long position.

**Comment by gbeced** [comment, marked as answer]
Not yet, but I'm working on supporting loans in the backtesting exchange: https://github.com/gbeced/basana/tree/feature/loans
